src/shared/pager.c

```diff
@@ -53,6 +53,9 @@ noreturn static void pager_fallback(void) {
         _exit(EXIT_SUCCESS);
 }
 
+static int store_stdout = 0;
```

Comment:
That name sounds like a conditional. Also, initialization to 0 (which is a valid fd) is wrong. It should be -1.